docs/using-gitbase/getting-started.md

```diff
@@ -0,0 +1,72 @@
+# Getting started
+
+## Prerequisites
+
+**gitbase** has two optional dependencies that should be running on your system if you're planning on using certain functionality.
+
+- [bblfsh](https://github.com/bblfsh/bblfshd) >= 2.5.0 (only if you're planning to use the `UAST` functionality provided in gitbase).
+- [pilosa](https://github.com/pilosa/pilosa) 0.9.0 (only if you're planning on using indexes).
+
+## Download an use the binary
+
+Check the [Release](https://github.com/src-d/gitbase/releases) page to download the gitbase binary.
+
+For more info about executable parameters, [go here](/docs/using-gitbase/configuration.md#executable-parameters).
+
+You can start a server by providing a path which contains multiple git repositories `/path/to/repositories` with this command:
+
+```
+$ gitbase server -v -g /path/to/repositories -u gitbase
+```
+
+## Installing from source
+
+Because gitbase uses [bblfsh's client-go](https://github.com/bblfsh/client-go), which uses cgo, you need to install some dependencies by hand instead of just using `go get`.
+
+_Note_: we use `go get -d` so the code is not compiled yet, as it would
```

Comment:
I rather use bold for **Note**. Not very fan of underlines in screens but this is a personal thing.